src/cmd/health.js

```diff
@@ -349,14 +366,21 @@ async get_storage_status(config_root, type, all_details) {
           invalid_storage = {
             name: config_data.name,
             storage_path: storage_path,
-            'code': 'STORAGE_NOT_EXIST',
+            code: health_errors.STORAGE_NOT_EXIST.error_code,
```

Comment:
we can have ENOENT error on the readFile(config_file_path) also right?
same for the other errors, we need to have a separate try catch for readFile(config_file) and ceckAccess(storage_path)

Reply:
@romayalon chances for getting `ENOENT` in conig file path is impossible since we consider only those files that are already there in the config root path. Missing read access for config file is something we can consider, WDYT?

Reply:
I agree it's not a common case, but it's possible if the bucket was deleted between the time we read the directory and this line, actually the read access is less worrying me because we run this script with sudo